pkgs/by-name/gh/ghostunnel/package.nix

```diff
@@ -17,29 +19,31 @@ buildGoModule rec {
     hash = "sha256-NnRm1HEdfK6WI5ntilLSwdR2B5czG5CIcMFzl2TzEds=";
   };
 
-  vendorHash = null;
+  vendorHash = "sha256-vP8OtjpYNMm1KkNfD3pmNrHh3HRy1GkzUbfLKWKhHbo=";
 
   deleteVendor = true;
 
-  # The certstore directory isn't recognized as a subpackage, but is when moved
-  # into the vendor directory.
-  postUnpack = ''
-    mkdir -p $sourceRoot/vendor/ghostunnel
-    mv $sourceRoot/certstore $sourceRoot/vendor/ghostunnel/
-  '';
+  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
+    apple-sdk_12
+    (darwinMinVersionHook "12.0")
+  ];
+
+  checkFlags = [ "-skip=^Test(ImportDelete|Signer|Certificate)(RSA|ECDSA|EC)$" ];
 
   passthru.tests = {
     nixos = nixosTests.ghostunnel;
     podman = nixosTests.podman-tls-ghostunnel;
   };
 
   meta = with lib; {
```

Comment:
Could you remove `with lib;`?

Reply:
I’d prefer to not change that here. It’s harmless and unrelated to what this PR is doing

Reply:
@yzx9 I gather that you're passionate about this, but many people don't care much about syntax-level issue, and giving them unnecessary and unrewarding work isn't really fair, but you could join existing efforts to decide, document and apply a code style.
If you're interested in improving your contributions to open source in general, I can recommend giving [Code review antipatterns by Simon Tatham](https://www.chiark.greenend.org.uk/~sgtatham/quasiblog/code-review-antipatterns/) a read (~10 minute).

Reply:
@roberth Thanks for the suggestion! I appreciate it, and I’ll definitely give it a read.